nautobot/extras/utils.py

```diff
@@ -391,6 +391,19 @@ def get_worker_count(request=None, queue=None):
     return celery_queues.get(queue, 0)
 
 
+def get_job_queue_worker_count(request=None, job_queue=None):
+    """
+    Return a count of the active Celery workers in a specified queue. Defaults to the `CELERY_TASK_DEFAULT_QUEUE` setting.
+    Same as get_worker_count() method above, but job_queue is an actual JobQueue model instance.
+    """
+    celery_queues = get_celery_queues()
```

Comment:
This is mildly expensive isn't it? Do we need some caching here so that we're not repeating this lookup N times if we have N defined queues?

Reply:
Understood, can you point me to a place where it is appropriate to clear the cache?

Reply:
I think in this case just setting it with a short timeout (say 5 seconds) is probably adequate.